package.json

```diff
@@ -77,6 +77,7 @@
     "chalk": "4.1.1",
     "chokidar": "^3.4.2",
     "cookie": "^0.4.2",
+    "graphql": "^15.0.0 || ^16.0.0",
```

Comment:
The only thing I'd like to verify is whether this won't get multiple `graphql` packages in an app that uses a different GraphQL version.

Reply:
is there a possibility to make a prerelease/canary version, which might be the easiest way to validate that?

I'm also not positive, if it does we can drop back to declaring this as a peer and dealing with the automatically installed peers behavior for projects that don't declare a local version

Reply:
@kettanaito 

To test, I published a version of this package to `@mattcosta7/msw`

I installed this package instead of `msw` in a project that consumes msw currently (without graphql included as a dependency - it was a transient dep by virtue of msw being in this pacakge already)

<img width="463" alt="Screen Shot 2022-08-30 at 10 28 37 PM" src="https://user-images.githubusercontent.com/8616962/187580033-c34bf212-20ef-4d23-8d90-0a86d47031e4.png">


I then decided that project should use `graphql@15.8`

<img width="306" alt="Screen Shot 2022-08-30 at 10 30 42 PM" src="https://user-images.githubusercontent.com/8616962/187579655-84ab47ba-ffbc-46e8-bd3d-11c5525bf277.png">

I think that should give a good indication that this de-duplication should happen on the package manager side

previously graphql _was_ in the pacakge-lock due to msw@0.44 

<img width="879" alt="Screen Shot 2022-08-30 at 10 33 24 PM" src="https://user-images.githubusercontent.com/8616962/187579925-fb7cf943-9774-4ea1-9749-254ead42b06e.png">

Reply:
Let me know if there are any other steps you'd like me to take to help verify this?

Reply:
That sounds great! Thank you for validating this, @mattcosta7. 

FIY, you can also install repositories at a specific PR state as

```json
{
  "msw": "mswjs/msw#1382/head"
}
```